
((digest . "3b628bd422bfb5752958fac31bdf7f77") (undo-list nil ("diff -r ./20tm/README.txt /home/ryuichi/work/ecu/20tm/README.txt
1c1
< 2018/07/26 \347\264\215\345\205\245
---
> 2018/09/14 \347\264\215\345\205\245
4,5c4,19
< \343\200\200ads_tyt_sils[Ver.006]
< \343\200\200ads_sils[Ver.009]
---
> \343\200\200ads_tyt_sils[Ver.009]
> \343\200\200ads_sils[Ver.d012]
> 
> \342\227\2062018/09/07\343\201\250\343\201\256\345\267\256\345\210\206
> \343\200\2005\346\234\253\347\211\210 ADS Ready\346\210\220\347\253\213\345\257\276\345\277\234
> \343\200\200 - v007\343\201\247\343\201\256BSW\351\203\250\345\257\276\345\277\234\343\202\222\343\203\236\343\203\274\343\202\270
> \343\200\200 - \345\234\260\345\233\263\343\203\207\343\203\274\343\202\277\344\273\225\346\247\230\345\244\211\346\233\264\345\257\276\345\277\234
> \343\200\200 - \343\202\253\343\203\241\343\203\251\343\203\221\343\203\251\343\203\241\343\203\274\343\202\277\345\200\244\345\233\272\345\256\232\345\214\226\345\257\276\345\277\234
>  
> \343\200\200Redmine\343\203\201\343\202\261\343\203\203\343\203\210\345\257\276\345\277\234
> \343\200\200 - ToDo #798
> \343\200\200 - ToDo #1087
> \343\200\200 - ToDo #1112 (pbuild\345\256\237\350\241\214\346\231\202\343\201\256\345\274\225\346\225\260\343\201\253 -ci \343\202\222\346\214\207\345\256\232)
> \343\200\200 
> \343\200\200\350\246\201\344\273\266\345\257\276\345\277\234
> \343\200\200 - 12.\343\203\255\343\202\260\343\202\222\350\246\213\343\201\237\343\201\250\343\201\215\343\201\253Error\343\202\204Warning\343\201\214\344\270\200\347\233\256\343\201\247\345\210\244\346\226\255\343\201\247\343\201\215\343\202\213
7,8d20
< \342\227\2062018/07/1\343\201\250\343\201\256\345\267\256\345\210\206
<   \343\203\273Ether/Can/\347\224\273\345\203\217\345\205\245\345\212\233\343\200\201\343\201\252\343\202\212\343\202\206\343\201\215\343\203\242\343\203\274\343\203\211\351\247\206\345\213\225\343\200\201SILS\350\244\207\346\225\260\350\265\267\345\213\225\343\200\201MDF4\345\207\272\345\212\233\343\200\201\343\202\277\343\202\271\343\202\257\343\202\263\343\202\242\350\250\255\345\256\232\345\257\276\345\277\234\343\200\201\344\273\256\346\203\263\346\231\202\351\226\223\346\223\215\344\275\234\343\200\201\343\202\263\343\203\263\343\203\225\343\202\243\343\202\260\343\203\225\343\202\241\343\202\244\343\203\253
10a23
> \343\200\200\343\203\273\343\203\200\343\202\244\343\202\242\343\202\260\347\204\241\345\212\271\345\214\226
11a25,26
> \343\200\200\343\203\273HTTP_client\347\204\241\345\212\271
> \343\200\200\343\203\273\347\224\273\345\203\217FFD\347\204\241\345\212\271
18,24c33,34
< \342\227\206\350\277\275\345\212\240\346\223\215\344\275\234(TMC\350\277\275\350\250\230)
< \343\200\200\347\224\273\345\203\217\343\203\207\343\203\274\343\202\277\343\201\257\343\202\265\343\202\244\343\202\272\343\201\214\345\244\247\343\201\215\343\201\204\343\201\237\343\202\201\343\200\201\346\234\254\343\203\252\343\203\235\343\202\270\343\203\210\343\203\252\343\201\256\347\256\241\347\220\206\345\244\226\343\201\250\343\201\227\343\201\246\343\201\204\343\201\276\343\201\231\343\200\202
< \343\200\200  000000.MLR
< \343\200\200  000001.MLR
< \343\200\200\343\201\256\343\203\225\343\202\241\343\202\244\343\203\253\343\202\222\344\270\213\350\250\230\343\201\256\343\201\204\343\201\232\343\202\214\343\201\213\343\201\256\345\205\261\346\234\211\343\203\225\343\202\251\343\203\253\343\203\200\343\201\213\343\202\211\343\202\263\343\203\224\343\203\274\343\201\227
< \343\200\200\343\200\200ads/data/image/
< \343\200\200\343\201\253\351\205\215\347\275\256\343\201\227\343\201\246\343\201\217\343\201\240\343\201\225\343\201\204\343\200\202
---
> \342\227\206\343\203\207\343\203\274\343\202\277\347\275\256\343\201\215\345\240\264(TMC\350\277\275\350\250\230)
> \343\200\200\347\224\273\345\203\217\343\203\207\343\203\274\343\202\277\343\200\201integdata\343\201\257\344\270\213\350\250\230\343\201\256\345\205\261\346\234\211\343\203\225\343\202\251\343\203\253\343\203\200\343\201\213\343\202\211\343\203\200\343\202\246\343\203\263\343\203\255\343\203\274\343\203\211\343\201\227\343\201\246\345\210\251\347\224\250\343\201\227\343\201\246\343\201\217\343\201\240\343\201\225\343\201\204\343\200\202
26,27c36,37
< \343\200\200\\\\jadskh004.cp.toyota.co.jp\\Pub\\\343\203\237\343\203\211\343\203\253\343\203\201\343\203\274\343\203\240\\ADS_SILS\\Software\\180726_ADS-SILS_ver006\\180726_ADS-SILS_ver006\\image_data
< \343\200\200\\\\172.16.7.76\\kudan01\\projects\\002_fpMW\351\226\213\347\231\272\\sec_tmc\345\205\261\346\234\211\\To_\346\243\256\346\247\230\\ADS-SILS-\344\270\200\346\231\202\345\205\261\346\234\211\\01_SILS\343\202\263\343\203\274\343\203\211\\180726_ADS-SILS_ver006\\image_data
---
> \343\200\200\\\\jadskh004.cp.toyota.co.jp\\Pub\\\343\203\237\343\203\211\343\203\253\343\203\201\343\203\274\343\203\240\\ADS_SILS\\Software\\180726_ADS-SILS_ver006\\1180914_ADS-SILS_ver009\\data
> \343\200\200\\\\172.16.7.76\\kudan01\\projects\\002_fpMW\351\226\213\347\231\272\\sec_tmc\345\205\261\346\234\211\\To_\346\243\256\346\247\230\\ADS-SILS-\344\270\200\346\231\202\345\205\261\346\234\211\\01_SILS\343\202\263\343\203\274\343\203\211\\180914_ADS-SILS_ver009\\data
./20tm/ads \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: .gitignore
" . -1) ((marker . 1) . -2056) ((marker . 1) . -1235) ((marker . 567) . -2056) ((marker . 567) . -2056) ((marker . 718) . -1299) ((marker . 718) . -1299) ((marker* . 1) . 2056) ((marker* . 1) . 2056) ((marker . 1) . -2056) ((marker . 745) . -1295) ((marker . 745) . -1295) ((marker . 775) . -1235) ((marker . 775) . -1235) ((marker . 663) . -1466) ((marker . 663) . -1466) ((marker . 628) . -1597) ((marker . 628) . -1597) ((marker . 619) . -1745) ((marker . 619) . -1745) ((marker . 598) . -1749) ((marker . 598) . -1749) ((marker . 594) . -1875) ((marker . 594) . -1875) ((marker . 573) . -2017) ((marker . 573) . -2017) ((marker) . -2056) ((marker . 691) . -1454) ((marker . 691) . -1454) ((marker . 712) . -1334) ((marker . 712) . -1334) ((marker . 749) . -1259) ((marker . 749) . -1259) ((marker* . 1) . 39) ((marker . 1) . -1235) ((marker . 1) . -2056) 2057 (t 23486 52474 581001 0) nil ("> 0c7cccce6388186597844e61280709945c172151 e7f36b60b1606a9841be1ff7d1aac45c892ca392 ryuichi.ando <ryuichi.ando@fixstars.com> 1539225712 +0900	cherry-pick: \351\200\201\344\277\241\343\203\242\343\202\270\343\203\245\343\203\274\343\203\253\343\203\227\343\203\255\343\203\210\343\202\277\343\202\244\343\203\227\350\277\275\345\212\240
> e7f36b60b1606a9841be1ff7d1aac45c892ca392 f857db0fb0842abd6f664926c5d3d233f0e9eb2b ryuichi.ando <ryuichi.ando@fixstars.com> 1539225956 +0900	cherry-pick: \345\217\227\344\277\241\343\203\242\343\202\270\343\203\245\343\203\274\343\203\253\343\203\227\343\203\255\343\203\210\343\202\277\343\202\244\343\203\227\350\277\275\345\212\240
> f857db0fb0842abd6f664926c5d3d233f0e9eb2b 90b89b739462f71708fa861c00c8bc4394fe30d1 ryuichi.ando <ryuichi.ando@fixstars.com> 1539225998 +0900	cherry-pick: \351\200\201\344\277\241\343\203\242\343\202\270\343\203\245\343\203\274\343\203\253\343\201\256\343\203\206\343\202\271\343\203\210\343\203\207\343\203\274\343\202\277\347\224\237\346\210\220\343\202\277\343\202\244\343\203\237\343\203\263\343\202\260\343\202\222\345\244\211\346\233\264
> 90b89b739462f71708fa861c00c8bc4394fe30d1 71d8d8c80091f504a3d7e6d66b20cb627084b414 ryuichi.ando <ryuichi.ando@fixstars.com> 1539226106 +0900	cherry-pick: \347\265\220\345\220\210\343\203\206\343\202\271\343\203\210\347\224\250\343\203\201\343\202\247\343\203\203\343\202\257\343\202\271\343\202\257\343\203\252\343\203\227\343\203\210\343\202\222\350\277\275\345\212\240
> 71d8d8c80091f504a3d7e6d66b20cb627084b414 cab8b7d21643c387724b3d5fa52b47cb856c2a8e ryuichi.ando <ryuichi.ando@fixstars.com> 1539226603 +0900	cherry-pick: select \343\202\222 poll\343\201\253\346\233\270\343\201\215\346\217\233\343\201\210
diff -r ./20tm/.git/logs/refs/heads/feature/appfw/ioapp_dnpf /home/ryuichi/work/ecu/20tm/.git/logs/refs/heads/feature/appfw/ioapp_dnpf
1c1
< 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539163187 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
---
> 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539224293 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
/home/ryuichi/work/ecu/20tm/.git/logs/refs/heads \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: make3_ioipp_dnpf2_324
diff -r ./20tm/.git/logs/refs/remotes/origin/HEAD /home/ryuichi/work/ecu/20tm/.git/logs/refs/remotes/origin/HEAD
1c1
< 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539163187 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
---
> 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539224293 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 05
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 0c
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 0e
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 16
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 1c
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 27
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 38
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 42
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 4a
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 4c
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 6f
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 71
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 74
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 7a
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 80
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 81
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 8c
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 90
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: 98
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: a5
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: b5
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: c8
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: c9
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: ca
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: d8
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: df
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: e0
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: e5
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: e7
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: f0
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: f3
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: f5
/home/ryuichi/work/ecu/20tm/.git/objects \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: f8
/home/ryuichi/work/ecu/20tm/.git/refs/heads \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: make3_ioipp_dnpf2_324
/home/ryuichi/work/ecu/20tm \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: .gitignore
" . -1) ((marker . 1) . -4402) ((marker . 1) . -2982) ((marker . 567) . -4019) ((marker . 567) . -4019) ((marker . 383) . -3653) ((marker . 383) . -3653) ((marker . 718) . -4080) ((marker . 718) . -4080) ((marker* . 1) . 4402) ((marker* . 1) . 4402) ((marker . 1) . -4402) ((marker . 389) . -3714) ((marker . 389) . -3714) ((marker . 453) . -3592) ((marker . 453) . -3592) ((marker) . -4402) ((marker . 242) . -3531) ((marker . 242) . -3531) ((marker . 119) . -3470) ((marker . 119) . -3470) ((marker . 337) . -3409) ((marker . 337) . -3409) ((marker . 745) . -4263) ((marker . 745) . -4263) ((marker . 775) . -4402) ((marker . 775) . -4402) 4403 nil ("/home/ryuichi/work/ecu/20tm \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: .clang-format
/home/ryuichi/work/ecu/20tm/.git \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: COMMIT_EDITMSG
diff -r ./20tm/.git/HEAD /home/ryuichi/work/ecu/20tm/.git/HEAD
1c1
< 2b884015829e2274c679f2c23d74d46888e1220e
---
> ref: refs/heads/make3_ioipp_dnpf2_324
/home/ryuichi/work/ecu/20tm/.git \343\201\256\343\201\277\343\201\253\345\255\230\345\234\250: ORIG_HEAD
diff -r ./20tm/.git/config /home/ryuichi/work/ecu/20tm/.git/config
11a12,14
> [branch \"make3_ioipp_dnpf2_324\"]
> 	remote = origin
> 	merge = refs/heads/make2_ioipp_dnpf2_324
\343\203\220\343\202\244\343\203\212\343\203\252\343\203\274\343\203\225\343\202\241\343\202\244\343\203\253 ./20tm/.git/index \343\201\250/home/ryuichi/work/ecu/20tm/.git/index \343\201\257\347\225\260\343\201\252\343\202\212\343\201\276\343\201\231
diff -r ./20tm/.git/logs/HEAD /home/ryuichi/work/ecu/20tm/.git/logs/HEAD
1,2c1,9
< 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539163187 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
< f9d5739b4d889dff87e5d02a342fe0435a2c7186 2b884015829e2274c679f2c23d74d46888e1220e ryuichi.ando <ryuichi.ando@fixstars.com> 1539230638 +0900	checkout: moving from feature/appfw/ioapp_dnpf to 2b884015
---
> 0000000000000000000000000000000000000000 f9d5739b4d889dff87e5d02a342fe0435a2c7186 ryuichi.ando <ryuichi.ando@fixstars.com> 1539224293 +0900	clone: from ssh://git@gl.fixstars.com:8022/nxtmc-adas/ECU-FW-group/20tm.git
> f9d5739b4d889dff87e5d02a342fe0435a2c7186 4ede0783a826c51c892ae0f9e56bf5fcc0d9564c ryuichi.ando <ryuichi.ando@fixstars.com> 1539224371 +0900	checkout: moving from feature/appfw/ioapp_dnpf to make3_ioipp_dnpf2_324
> 4ede0783a826c51c892ae0f9e56bf5fcc0d9564c bed3b31f7c2772620e9c25bb391338820a47bf3b ryuichi.ando <ryuichi.ando@fixstars.com> 1539224399 +0900	reset: moving to HEAD^
> bed3b31f7c2772620e9c25bb391338820a47bf3b 0c7cccce6388186597844e61280709945c172151 ryuichi.ando <ryuichi.ando@fixstars.com> 1539224543 +0900	commit (cherry-pick): cherry-pick 4f1fb02c
" . 1) ((marker . 1) . -1905) ((marker . 1) . -773) ((marker . 567) . -1905) ((marker . 567) . -1905) ((marker* . 1) . 1905) ((marker* . 1) . 1905) ((marker . 1) . -1905) ((marker . 389) . -1119) ((marker . 389) . -1119) ((marker . 453) . -773) ((marker . 453) . -773) ((marker) . -1905) 1906 (t 23486 52327 201001 0)))
